pylon/README.md

```diff
@@ -0,0 +1,25 @@
+# Pylon: Universal Entry to Inner APIs
+
+## Usage
+
+URL pattern:
+
+```
+http://<pylon_ip>:<pylon_port>/r/http/<api_server_ip>/<api_server_port>/...
+```
+
+For example:
+
+- Example 1 – Kubernetes api: http://10.151.40.179:8086/r/http/10.151.40.179/8080/api/v1/nodes
```

Comment:
It would be wonderful if this "/http/ip/port/" can be replaced with "/kubernetes/" so that caller can get a fixed URL whatever where we deployed kubernetes.

Reply:
test reply

Reply:
The same to other urls.

Reply:
Yes Bin, I totally agree. I plan to do this in the next step -- In this PR, we implement the basic functionality; and then in the next PR, we refine the interface. This is because the nginx-based reverse proxy is not the highest priority. How about this idea?

Reply:
Yes, that's what we discussed on the scrum meeting. Putting comments here is for reminding.

Reply:
Will do.